modin/config/__main__.py

```diff
@@ -62,12 +68,12 @@ def export_config_help(filename: str):
                 "Default Value": obj._get_default()
                 if obj.__name__ != "RayRedisPassword"
                 else "random string",
-                "Description": obj.__doc__,
+                "Description": description,
                 "Options": obj.choices,
             }
-            configs = configs.append(data, ignore_index=True)
```

Comment:
`DataFrame.append` method is deprecated and should be removed.

Reply:
Where do we use it? configs_data is a list

Reply:
Yes, `configs_data` is a list, so we can use `append` method without pandas deprecation warnings (replaced `configs` is `pandas.DataFrame`).